recipe/meta.yaml

```diff
@@ -297,25 +284,31 @@ outputs:
         - wheel
       run:
         - python
-        - joblib <2,>=1.1
-        - numpy <1.29,>=1.21
-        - scipy <1.13,>=1.5.4
-        - pandas <2.2.0,>=2.0.0
-        - pytorch <2.1,>=2.0
-        - lightning <2.1,>=2.0.0
-        - pytorch-lightning <2.1,>=2.0.0
-        - statsmodels <0.15,>=0.13.0
-        - gluonts <0.15,>=0.14.0
-        - networkx <4,>=3.0
-        - statsforecast <1.5,>=1.4.0
-        - mlforecast <0.10.1,>=0.10.0
-        - utilsforecast <0.0.11,>=0.0.10
-        - tqdm <5,>=4.38
-        - orjson >=3.9,<3.10
-        - tensorboard <3,>=2.9
-        - {{ pin_subpackage(name ~ '.core', exact=True) }}
-        - {{ pin_subpackage(name ~ '.common', exact=True) }}
-        - {{ pin_subpackage(name ~ '.tabular', exact=True) }}
+        - joblib >=1.1,<2
+        - lightning {{ lightning }}
+        - pytorch-lightning {{ lightning }}
+        ## transformers[sentencepiece] ##
+        - transformers {{ transformers }}
+        - sentencepiece {{ sentencepiece }}
+        - protobuf
+        ## transformers[sentencepiece] ##
+        - huggingface_accelerate {{ huggingface_accelerate }}
+        - gluonts ==0.15.1
+        - statsforecast >=1.4.0,<1.5
+        - mlforecast >=0.10.0,<0.10.1
+        - utilsforecast >=0.0.10,<0.0.11
+        - orjson >=3.9,<4
+        - tensorboard >=2.9,<3
+        - {{ pin_subpackage('autogluon.core', exact=True) }}
+        - {{ pin_subpackage('autogluon.common', exact=True) }}
+        - {{ pin_subpackage('autogluon.tabular', exact=True) }}
+        # Extra group "all". autogluon requires autogluon.timeseries[all].
+        - optimum >=1.17,<1.19
+        - onnx
```

Comment:
I don't see a dependency on `onnx`

Reply:
I included this block to build the `[all]` extra for `autogluon.timeseries`, which is required for `autogluon`. This is why we have `onnx`, `datasets`, etc..

Reply:
can you point me to the requirement? I still don't see it.

Reply:
[This](https://github.com/autogluon/autogluon/blob/65d1430c4f8fa3803bc57e8e978374e7ff32a982/timeseries/setup.py#L62) specifies `optimum[onnxruntime]`, which entails these [packages](https://inspector.pypi.io/project/optimum/1.17.0/packages/fe/67/6301d17c9ee28db378c766355cf43a8054791eda568a498391bf59913fea/optimum-1.17.0.tar.gz/optimum-1.17.0/PKG-INFO#line.31).